### PR TITLE
Add 0xDollar space

### PR DESCRIPTION
**Pull request by 0xFranklin**
Add 0xDollar space to snapshot

### CODE REVIEW

**Review by e-asia — COMMENTED** [review]
The images file size should not exceed 50KB. Please reduce their weight.

**Comment by 0xFranklin** [comment]
@e-asia I compress the png. Thx for reminding

File: spaces/0xdollar/index.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "name": "0xDollar.Finance",
+  "defaultView": "core",
+  "address": "0x34870322f473830d28d28bef7f1d4178edf832d8",
+  "token": "0x34870322f473830d28d28bef7f1d4178edf832d8",
+  "core": [
+    "0xaE0091bD0689c32c38e062C6956c2240eC658bE4"
+  ],
+  "min": 0,
+  "invalid": []
+}
```
Comment (e-asia) [inline spaces/0xdollar/index.json:13]: We have updated the format in `index.json`. 
Please add 2 new parameters `chainId` and `decimals`.
You can refer to the documentation that has been completed. https://docs.snapshot.page/guides/create-a-space#3-change-your-space-metadata

**Review by e-asia — CHANGES_REQUESTED** [review]

**Comment by 0xFranklin** [comment]
@e-asia I already add chainId and decimals. Thx

**Review by e-asia — APPROVED** [review]

**Pull request merged**